.github/actions/generate-metadata/action.yml

```
@@ -56,7 +56,7 @@ runs:
           env:
             GITHUB_TOKEN: ${{ inputs.gh_token }}
           run: |
-            aiida-registry fetch
+            aiida-registry fetch aiida-kkr aiida-fleur aiida-gudhi aiida-defects
```

is this a leftover from testing?

---

Yes, I forgot to change it back, just to explicitly list to make the test fast.
```suggestion
            aiida-registry fetch
```